docs/adr/0010-bdd-testing.md

```diff
@@ -0,0 +1,19 @@
+# 10. BDD testing
+
+Date: 2020-10-25
+
+## Status
+
+Pending
+
+## Context
+
+Behavior-driven development (BDD) allows for a broader team to collaborate on software development. The business stakeholders have insight into how well the software meets the requirements. Through automated tests they have a way to validate the functionality in a user friendly way. Cucumber, and it particular cucumber-js was selected as the tool of choice to accomplish this. To go along with cucumber-js, puppeteer as well as selenium webdriver were looked at to provide a full solution for BDD testing.
```

Comment:
> Cucumber, and it particular cucumber-js was selected as the tool of choice to accomplish this

Feels like this should be in the `Decision` section and is missing the selection of puppeteer.

Reply:
Good point. The idea was to present why we chose Puppeteer, since it was compared to some extent with the selenium webdriver. I didn't look at alternatives to Cucumber itself. I did reorganize the sections though to hopefully make it clearer. Hope it makes more sense now.